bom/pom.xml

```diff
@@ -853,7 +853,7 @@
             </dependency>
             <dependency>
                 <groupId>io.smallrye</groupId>
-                <artifactId>smallrye-health</artifactId>
+                <artifactId>smallrye-health-2.0</artifactId>
```

Comment:
`smallrye-health-2.0`?? Why do we use version in the artifactId?

Reply:
It's the new versioning that @kenfinnigan introduced [here](https://groups.google.com/d/msg/smallrye/xs5HFiTq6gk/90pShrJXBgAJ).

Reply:
Right, it signifies the MP version it aligns with

Reply:
Ok. I don't like it. But it's my fault that I missed the discussion. It's IMHO even more confusing if a GAV contains multiple versions.